### PR TITLE
Added event filter UI

### PR DESCRIPTION
**Pull request by nigow**
### Ticket
[Ticket Link](https://trello.com/c/jNpQzPMO/33-event-page-create-events-filter-bar)

### What is changed / added
- Added filters
- No internal logic added

### Screenshots
Screen Shot 2021-11-21 at 4 42 31

### CODE REVIEW

File: components/Common/EventFilter.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+const OptionButton: FC<OptionButtonProps> = ({ optionName } : OptionButtonProps) => {
+  const [clicked, setClicked] = useState(false);
+  const handleClick = () => {
+    setClicked(!clicked);
+  };
+
+  return (
+    <button type="button" onClick={handleClick} className="m-5">
```
Comment (TheFatPanda97) [inline components/Common/EventFilter.tsx:18]:
```suggestion
    <button type="button" onClick={() => setClicked(!clicked)} className="m-5">
```

File: components/Common/EventFilter.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+interface OptionButtonProps {
+  optionName: string
+}
+
+const OptionButton: FC<OptionButtonProps> = ({ optionName } : OptionButtonProps) => {
+  const [clicked, setClicked] = useState(false);
+  const handleClick = () => {
+    setClicked(!clicked);
+  };
```
Comment (TheFatPanda97) [inline components/Common/EventFilter.tsx:15]:
```suggestion
```
we can make this inline

File: components/Common/EventFilter.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+  return (
+    <button type="button" onClick={handleClick} className="m-5">
+      {clicked
+        ? <p style={{ color: '#7d25af' }}>{optionName}</p>
+        : <p style={{ color: '#7d7b76' }}>{optionName}</p>}
```
Comment (TheFatPanda97) [inline components/Common/EventFilter.tsx:21]:
```suggestion
        <p style={{ color: clicked ? '#7d25af' :  '#7d7b76' }}>{optionName}</p>
```

**Review by TheFatPanda97 — CHANGES_REQUESTED** [review]
looks good, just small changes here and there

**Comment by nigow** [comment]
@TheFatPanda97 ok

**Review by TheFatPanda97 — APPROVED** [review]

**Pull request merged**